core/whistle_number_manager-1.0.0/include/wh_number_manager.hrl

```diff
@@ -73,7 +73,7 @@
                                        ,<<"failover">> ,<<"prepend">>
                                       ]).
 
--define(WNM_DB_PREFIX_L, "numbers/").
+-define(WNM_DB_PREFIX_L, "numbers/+").
```

Comment:
@lazedo true but...still makes me nervous... if it is just for an efficiency with managed numbers I would prefer to remove it.  I think that carrier module is only used on dev boxes...